.env.example

```diff
@@ -1 +1,2 @@
 NODE_ENV=development
+ENABLE_REDUX_DEVTOOLS=true
```

Comment:
does this do something with zustand? just an artifact?

Reply:
There's a zustand middleware that integrates with the redux-devtools browser extension and I just added this env variable myself (which is read in `src/config/index.ts`) to control whether it's enabled or not.